src/mocks/data.ts

```diff
@@ -4792,3 +4792,60 @@ export const mockFragment = (pageId: string) => ({
     ],
     published: true
 });
+
+export const mockPage = (pageId: string) => ({
+    id: pageId,
+    type: "page",
+    canonicalSourceFile: `content/mocks/${pageId}.json`,
+    encoding: "markdown",
+    title: `Mock page: ${pageId}`,
+    children: [
+      {
+        type: "content",
+        value: "Here is some mock content for you!",
+        encoding: "markdown",
+        published: true
+      }
+    ],
+    published: true
+  });
+
+  export const mockAssignedQuiz = [{
+    id: 1,
+    quizId: "example_quiz_1",
+    quizSummary: {
+        id: "example_quiz_1",
+        title: "Example Quiz 1",
+        type: "isaacQuiz",
+        level: 2,
+        tags: [],
+        url: "/isaac-api/api/quiz/example_quiz_1",
+        visibleToStudents: false},
+    groupId: 1,
+    ownerUserId: mockUser.id,
+    creationDate: 888888888888 as unknown as Date,
+    dueDate: 999999999999 as unknown as Date,
+    quizFeedbackMode: "NONE"
+    }];
+
+    export const mockAvailableQuiz = { 
+        results: [
+            {   id: "example_quiz_1",
+                title: "Example Quiz 1",
+                type: "isaacQuiz",
+                level: "2",
+                tags: [],
+                url: "/isaac-api/api/quiz/example_quiz_1",
+                visibleToStudents: false
+            },
+            {   id: "example_quiz_2",
```

Comment:
The indentation and style of what has been added seems a bit off. I have created a [new ticket](https://github.com/isaaccomputerscience/isaac-cs-issues/issues/25) to add the linter to the pipeline to ensure that the code is consistent. However, I think there are many linter issues across the whole codebase, so I am going to open another ticket to fix the linter issues as well. You don't need to fix these issues as part of your ticket, but at least all the code that you add or change should comply with the linter rules. If you install the ESLint extension in your IDE, it should highlight any errors or warnings, as well as offer you the ability to automatically fix the whole file.

Reply:
Ah, it looks like there is already a ticket to fix the linting errors: https://github.com/isaaccomputerscience/isaac-cs-issues/issues/8

Reply:
Made some manual adjustments, but as discussed we will need to address existing linting errors, linting rules, and possible addition of prettier in other tickets, as current ESLint config is not picking up on indentation or semicolons etc.